public/index.html

```diff
@@ -292,6 +292,7 @@ <h3>Your aggregation strategy</h3>
                                 <div class="input-hint">⚠️ The total contribution budget is split across measurement
                                     goals following the percentages/values below, so the sum of all percentages/values can't exceed
                                     100/total contribution budget.</div>
+                                <div class="input-hint" id="help-budget-split-warning" hidden> ⚠️ Because you're using Key Strategy 'B' the total contribution budget is now evenly split between all sub-keys. </div>
```

Comment:
```suggestion
                                <div class="input-hint" id="help-budget-split-warning" hidden> ⚠️ Because you're using Key Strategy 'B' the total contribution budget is now evenly split between all sub-keys. Note that the total number of sub-keys is: (number of key structures per measurement goal) * (number of measurement goals). </div>
```